CONTRIBUTING.md

```diff
@@ -688,9 +688,10 @@ Unless:
             # Create a new environment with new cursor database
             new_env = api.Environment(new_cr, self.env.uid, self.env.context)
             # with_env replace original env for this method
+            # A good comment here of why this isolated transaction is required.
             self.with_env(new_env).write({'name': 'hello'})  # isolated transaction to commit
-            new_env.cr.commit()  # Don't show a invalid-commit in this case
         # You don't need close your cr because is closed when finish "with"
```

Comment:
Instead of putting 2 lines, rephrase this one as:
```
# You don't need to close nor to commit your cursor as they are done when exiting "with" block
```